isaric/parsers/isaric-ecmo.toml

```diff
@@ -1168,6 +1174,49 @@
   date = { field = "daily_dsstdat" }
   value = { field = "daily_temp_vsorres", unit = "°C", source_unit = { field = "daily_temp_vsorresu", values = { 1 = "°C", 2 = "°F" } } }
 
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { ref = "dailyDateHierarchy" }
+  value = { field = "daily_fio2b_lborres", apply = { function = "Percentage" } }
+
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { ref = "dailyDateHierarchy" }
+  value = { field = "daily_fio2_lborres", apply = { function = "Percentage" } }
+
+[[observation]]
+  name = "oxygen_flow_rate"
+  phase = "study"
+  date = { ref = "dailyDateHierarchy" }
+  value = { field = "daily_fio2c_lborres" }
+
+  [observation.context]
+    combinedType = "set"
+    excludeWhen = "none"
+    fields = [
+      { field = "daily_nasaloxy_cmtrt", values = { 1 = "HFNC" } },
+      { field = "daily_noninvasive_prtrt", values = { 1 = "non-invasive" }, if = { "any" = [
+        { daily_noninvasive_prtype = "" },
+        { daily_noninvasive_prtype = 4 },
+      ] } },
+      { field = "daily_noninvasive_prtype", values = { 1 = "BiPAP", 2 = "CPAP", 3 = "Other non-invasive" } },
+      { field = "daily_invasive_prtrt", values = { 1 = "invasive mechanical" } },
+      { field = "daily_ecmo_prtrt", values = { 1 = "ECMO" }, if = { "any" = [
+        { daily_ecmo_prtype = "" },
+        { daily_ecmo_prtype = 4 },
+      ] } },
+      { field = "daily_ecmo_prtype", values = { 1 = "VV ECMO", 2 = "AV ECMO", 3 = "Central ECMO" } },
+    ]
```

Comment:
i think it might be best to drop the context here, as this data doesn't appear to have been collected in direct relation to the fio2 values, so we don't know whether the fio2 value given relates to one or any of the oxygen therapy types

Reply:
Do you mean the whole context section, or just the daily_ecmo_prtrt section?

Reply:
@pipliggins I think the whole context section should be dropped as no specific device was indicated alongside the FiO2 values